src/views/Admin.js

```diff
@@ -227,23 +247,23 @@ export default class AdminPanel extends Component {
     return !Array.isArray(this.state.users)
       ? null
       : this.getFilteredUsers()
-        .slice(
-          this.state.usersPage * this.state.usersRowsPerPage,
-          this.state.usersPage * this.state.usersRowsPerPage +
-          this.state.usersRowsPerPage
-        )
-        .map((currentuser) => {
-          return (
-            <User
-              user={currentuser}
-              nextPath={this.nextPath}
-              changeRole={this.changeRole}
-              showModal={this.showDeleteUserModal}
-              role={this.role}
-              key={currentuser._id}
-            />
-          );
-        });
+          .slice(
+            this.state.usersPage * this.state.usersRowsPerPage,
+            this.state.usersPage * this.state.usersRowsPerPage +
+              this.state.usersRowsPerPage
+          )
+          .map((currentuser) => {
+            return (
+              <User
+                user={currentuser}
+                nextPath={this.nextPath}
+                changeRole={this.changeRole}
+                showModal={this.showDeleteUserModal}
+                role={this.role}
+                key={currentuser._id}
+              />
+            );
+          });
   }
 
```

Comment:
Not exactly part of this PR but the filter panel is overlapping the table as the page collapses:
![Screen Shot 2021-11-02 at 11 37 36 AM](https://user-images.githubusercontent.com/13375042/139880248-2f1a09f6-9ef0-43bf-9ed1-a17dcfb4ce8e.png)